packages/usa-combo-box/src/styles/_usa-combo-box.scss

```diff
@@ -56,13 +56,23 @@ $close: map-merge(
   margin-bottom: 0;
   max-width: none;
   padding-right: calc(2.5em + 3px);
+
+  &:disabled,
+  &[aria-disabled="true"] {
+    @include u-disabled;
+  }
```

Comment:
Wouldn't adding disabled styles to `%block-input-styles` make it easier to style or does that cause issues with other components?

Reply:
Has a negative effect on `usa-range` component because it also uses `%block-input-styles`.

https://github.com/uswds/uswds/blob/c662f50928025ed1dc8c40e04b258e7f879cfb17/packages/usa-range/src/styles/_usa-range.scss#L53